Code/Tools/AssetProcessor/native/ui/CacheServerData.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <native/utilities/AssetUtilEBusHelper.h>
+#include <native/utilities/PlatformConfiguration.h>
+#include <AzCore/IO/Path/Path.h>
+#endif
+
+namespace AssetProcessor
+{
+    struct CacheServerData
+    {
+        enum class ErrorLevel
```

Comment:
We define our own error levels here over AZ Trace etc?

Reply:
This is GUI Error Level, not logging error level